src/CommunityToolkit/Xamarin.CommunityToolkit/Effects/StatusBar/StatusBarEffect.shared.cs

```diff
@@ -0,0 +1,53 @@
+﻿using System.Linq;
+using Xamarin.Forms;
+
+namespace Xamarin.CommunityToolkit.Effects
+{
+	public class StatusBarEffect : RoutingEffect
+	{
+		public static readonly BindableProperty ColorProperty = BindableProperty.CreateAttached(
+			nameof(Color), typeof(Color), typeof(StatusBarEffect), Color.Default, propertyChanged: TryGenerateEffect);
+
+		public static readonly BindableProperty StyleProperty = BindableProperty.CreateAttached(
+			nameof(Style), typeof(StatusBarStyle), typeof(StatusBarEffect), StatusBarStyle.Default, propertyChanged: TryGenerateEffect);
+
+		public StatusBarEffect()
+			: base(EffectIds.StatusBar)
+		{
+			#region Required work-around to prevent linker from removing the platform-specific implementation
+#if __ANDROID__
+			if (System.DateTime.Now.Ticks < 0)
+				_ = new Android.Effects.PlatformStatusBarEffect();
+#elif __IOS__
+			if (System.DateTime.Now.Ticks < 0)
+				_ = new iOS.Effects.PlatformStatusBarEffect();
+#elif UWP
+			if (System.DateTime.Now.Ticks < 0)
+				_ = new UWP.Effects.PlatformStatusBarEffect();
+#endif
+			#endregion
+		}
+
+		public static Color GetColor(BindableObject bindable) =>
+			(Color)bindable.GetValue(ColorProperty);
+
+		public static StatusBarStyle GetStyle(BindableObject bindable) =>
+			(StatusBarStyle)bindable.GetValue(StyleProperty);
+
+		static void TryGenerateEffect(BindableObject bindable, object oldValue, object newValue)
+		{
+			if (bindable is not Page page)
+				return;
+
+			var oldEffect = page.Effects.FirstOrDefault(e => e is StatusBarEffect);
+			if (oldEffect != null)
+				page.Effects.Remove(oldEffect);
+
+			page.Effects.Add(new StatusBarEffect());
+		}
+
+		public Color Color => GetColor(Element);
+
+		public StatusBarStyle Style => GetStyle(Element);
```

Comment:
Let's remove those properties.
```suggestion
```

Reply:
This might break something. Let me check

Reply:
Didn't break, as I said I tested all those changes on my end.